src/parthenon_arrays.hpp

```diff
@@ -114,52 +114,27 @@ class ParArrayND {
     return GetDim(1)*GetDim(2)*GetDim(3)*GetDim(4)*GetDim(5)*GetDim(6)*sizeof(T);
   }
 
-  KOKKOS_INLINE_FUNCTION
-  T &operator() (const int n) {
-    return d6d_(0,0,0,0,0,n);
-  }
-  KOKKOS_INLINE_FUNCTION
+  KOKKOS_INLINE_FUNCTION __attribute__((always_inline))
```

Comment:
For these why not use `KOKKOS_FORCEINLINE_FUNCTION`?

Reply:
Because i didn’t know it existed. Good catch. I’ll fix it.